interop/tapir/src/main/scala/caliban/interop/tapir/HttpInterpreter.scala

```diff
@@ -31,6 +35,24 @@ sealed trait HttpInterpreter[-R, E] { self =>
     endpoints[S](streams).map(_.serverLogic(logic(_)))
   }
 
+  def serverEndpointsFuture[S](streams: Streams[S])(runtime: Runtime[R])(implicit
+    streamConstructor: StreamConstructor[streams.BinaryStream]
+  ): List[ServerEndpoint[S, Future]] = {
+    implicit val r: Runtime[R] = runtime
+    serverEndpoints(streams).map(
+      convertHttpEndpointToFuture[R, streams.BinaryStream, S, (GraphQLRequest, ServerRequest)]
+    )
+  }
+
+  def serverEndpointsIdentity[S](streams: Streams[S])(runtime: Runtime[R])(implicit
+    streamConstructor: StreamConstructor[streams.BinaryStream]
+  ): List[ServerEndpoint[S, Identity]] = {
+    implicit val r: Runtime[R] = runtime
+    serverEndpoints(streams).map(
+      convertHttpEndpointToIdentity[R, streams.BinaryStream, S, (GraphQLRequest, ServerRequest)]
+    )
+  }
```

Comment:
Should we make these methods package-private so that we don't have to worry about mima in the future? Or do we need them public?

Reply:
They need to be public, for anyone who want to use another adapter than the one we provide (e.g. ox).